corehq/apps/fixtures/upload/failure_messages.py

```diff
@@ -18,6 +18,10 @@
         "Lookup-tables should have unique 'table_id'. "
         "There are two rows with table_id '{tag}' in 'types' sheet."
     ),
+    "invalid_table_id": ugettext_noop(
+        "Lookup-tables 'table_id' must be valid XML identifiers."
```

Comment:
Minor, but I'd remove this line, it doesn't add info and many (most?) users will just be confused by the mention of XML.